pkg/postage/service.go

```diff
@@ -33,7 +33,7 @@ var (
 type Service interface {
 	Add(*StampIssuer) error
 	StampIssuers() []*StampIssuer
-	GetStampIssuer([]byte) (*StampIssuer, error)
+	GetStampIssuer([]byte) (*StampIssuer, func() error, error)
```

Comment:
Preferably `Service` should expose `Save` function and `GetStampIssuer` should retain old signature. Because returned `save` function of `GetStampIssuer` is not uniquely created on each call  (of `GetStampIssuer`) therfore if this function doesn't do anything specific to this call is should not be returned by this method.

Reply:
I prefer this pattern of returning a "cleanup" or "done" callback because it forces the user to use it. 
The exposed `Save` does not enforce this.

Reply:
How is returning this function forcing user to use it? In this PR it appears that new function signature actually creates dogsled in some places.

Reply:
Outside of tests, I don't see it anywhere else.